### ISSUE TITLE
Berry Boot USB .IMG issues

### ISSUE THREAD

**Issue by Vadero66** [maxnet/berryboot#731]
I have a raspberry pi 4 8gb model B. I am running berry boot. When I select my USB to find the flavor of OS I want to install, it displays nothing but system volume. I jave use pi images ro format 4 USB devices of different manufacturers.  It still won't display any .img files.

**Comment by symbios24** [comment]
Hi, The drive must be fat32 formatted to see it.

**Comment by Vadero66** [comment]
Yes it is fat 32 and formatted by the pi imaging tool. I have tried a 32gb
64gb 128gb USB drive. All images are squashed.

**Comment by symbios24** [comment]
Press the add os option and keep the left click pressed until you see the add from usb option

**Comment by Vadero66** [comment]
I have done this and it shows my USB drive. It also shows my file system
within the drive. I can create multiple sub folders and they all display
correctly on the berry boot menue.  The problem is that the image files do
not display. I have tried 10 different images and non display.

**Comment by symbios24** [comment]
Are the images with the img extension?

Eg:. Raspbian.img

Or you put them compressed in the drive?

Eg: Raspbian.img.gz

If they are in capital.IMG put them in lower case .img

**Comment by Vadero66** [comment]
Yes

**Comment by bernermic** [comment]
Hi, first of all - thanks for this great tool!

@Vadero66 I ran into the same exact issue. I was able to restore my .img from USB via accessing the `Backup` section.
It was the last option where I could choose to restore a img from USB, there it worked just fine for me.

Somehow the `add OS` option didn't show me a the restore from USB either.

**Comment by maxnet** [comment]
>Somehow the add OS option didn't show me a the restore from USB either.

Need to hold down your mouse button longer over the "add OS" button.

**Comment by bernermic** [comment]
Ok, I will try that with my next install.
